source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -466,6 +503,10 @@ let parse source_jsons =
   let lineage_analysis = List.exists ~f:parse_lineage_analysis source_jsons in
   let rules = List.rev_append rules generated_combined_rules in
   let matching_sources, matching_sinks = matching_kinds_from_rules rules in
+  if !unknown_keys_count != 0 then
+    Log.error
+      "%d unknown keys were found in the taint.config file(s) and will be ignored."
+      !unknown_keys_count;
```

Comment:
What is the objective with this additional message? We already say `Unknown key X encountered in section Y`. Could we just say that it will be ignored there? I would try to avoid mutable variables if possible.

Reply:
>  I would try to avoid mutable variables if possible.
Sure let's remove it :)